mmv1/templates/terraform/examples/access_context_manager_access_policy_scoped.tf.erb

```diff
@@ -0,0 +1,5 @@
+resource "google_access_context_manager_access_policy" "<%= ctx[:primary_resource_id] %>" {
+  parent = "organizations/123456789"
+  title  = "my policy"
+  scopes = ["<%= ctx[:vars]['scope'] %>"]
```

Comment:
Rather than plug in a scopes value, let's create a project in this configuration, reference it here (specifically using a Terraform reference i.e. `google_project.my_project.project_id` or equivalent for #, and then use it for the scoping project. That way, we'll ensure uniqueness across tests.